TestResultSummaryService/parsers/BenchmarkMetric.js

```diff
@@ -1,63 +1,87 @@
-// Note: The correct value for the metric must always reside in indice 1 (second element) of the regex output array
+// Note: The correct value for the metric must always reside in indice 1 (capture group) of the regex output array
+// use outerRegex when there are any warmup/cold runs before measure run that you do not wish to include.
+// if not familiar with regex look at other benchmarks & use https://regex101.com/ :)
 
```

Comment:
Let's add some convention here that would make it easier to add and debug regexes. For example, let's always use `\d` instead of `[0-9]` and `\.?` (not always a decimal) instead of `\.+` (always a decimal). This would avoid bugs like `(\d+.\d+)`, which won't catch 2 digit numbers such as 89 since it expects 3 digits at least.